data/lm/generate_lm.py

```diff
@@ -133,23 +156,31 @@ def main():
         description="Generate lm.binary and top-k vocab for DeepSpeech."
     )
     parser.add_argument(
-        "--input_txt",
-        help="Path to a file.txt or file.txt.gz with sample sentences",
+        "--input_txt", "-i",
```

Comment:
why not instead having a group options with pairs of `input_txt` and `top_k` ? As much as I remember, python argparse allows for that: https://pymotw.com/3/argparse/#argument-groups

Reply:
Okay, I'll look into this. So the idea is that we pair up an input_txt and a top_k for however many times the user needs them?

Reply:
Ineed, and so we delegate checking consistency to ArgParse instead of us :)

Reply:
(and it makes cleaner for the user)